Can't build main-eisop

I'm getting a failure to build the main-eisop branch on a brand-new clone.

```
$ ls -a
total 8
4 ./  4 ../
$ git clone git@github.com:jspecify/jspecify-reference-checker.git
Cloning into 'jspecify-reference-checker'...
remote: Enumerating objects: 4032, done.
remote: Counting objects: 100% (660/660), done.
remote: Compressing objects: 100% (249/249), done.
remote: Total 4032 (delta 375), reused 520 (delta 291), pack-reused 3372 (from 1)
Receiving objects: 100% (4032/4032), 996.26 KiB | 29.30 MiB/s, done.
Resolving deltas: 100% (1426/1426), done.
$ cd jspecify-reference-checker/
$ git checkout main-eisop 
branch 'main-eisop' set up to track 'origin/main-eisop'.
Switched to a new branch 'main-eisop'
$ ./gradlew build
git clone --depth 1 --single-branch https://github.com/jspecify/jdk.git ../jdk 
Cloning into '../jdk'...
Updating files: 100% (69213/69213)
Updating files: 100% (69213/69213), done.

> Task :compileJava FAILED
/usr/local/google/home/dpb/dev/jspecify-eisop/jspecify-reference-checker/src/main/java/com/google/jspecify/nullness/NullSpecAnnotatedTypeFactory.java:110: error: cannot find symbol
import org.checkerframework.framework.util.NoContractsFromMethod;
                                          ^
  symbol:   class NoContractsFromMethod
  location: package org.checkerframework.framework.util
/usr/local/google/home/dpb/dev/jspecify-eisop/jspecify-reference-checker/src/main/java/com/google/jspecify/nullness/NullSpecAnnotatedTypeFactory.java:1038: error: cannot find symbol
    return new NoContractsFromMethod();
               ^
  symbol:   class NoContractsFromMethod
  location: class NullSpecAnnotatedTypeFactory
/usr/local/google/home/dpb/dev/jspecify-eisop/jspecify-reference-checker/src/main/java/com/google/jspecify/nullness/NullSpecChecker.java:72: error: method does not override or implement a method from a supertype
  @Override
  ^
3 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
> Run with --info option to get more log output.
> Run with --scan to get full insights.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.10/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 32s
1 actionable task: 1 executed
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -19,11 +19,12 @@ jobs:
         with:
           repository: jspecify/jspecify
           path: jspecify
-      - name: Check out eisop/checker-framework
-        uses: actions/checkout@v4
-        with:
-          repository: eisop/checker-framework
-          path: checker-framework
+# Clone the EISOP CF if necessary:
+#      - name: Check out eisop/checker-framework
+#        uses: actions/checkout@v4
+#        with:
+#          repository: eisop/checker-framework
+#          path: checker-framework
       - name: Set up Java
         uses: actions/setup-java@v4
         with:
@@ -32,7 +33,9 @@ jobs:
       - name: Set up Gradle
         uses: gradle/gradle-build-action@v3
       - name: Build and Test
-        run: ./gradlew build conformanceTests demoTest --include-build ../jspecify --include-build ../checker-framework
+        run: ./gradlew build conformanceTests demoTest --include-build ../jspecify
+# If a cloned EISOP CF is needed, use the following:
+#        run: ./gradlew build conformanceTests demoTest --include-build ../jspecify --include-build ../checker-framework
         env:
           SHALLOW: 1
           JSPECIFY_CONFORMANCE_TEST_MODE: details

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -47,7 +47,9 @@ jobs:
         working-directory: jspecify
       - name: Run Samples Tests
         if: always()
-        run: ./gradlew jspecifySamplesTest --include-build ../jspecify --include-build ../checker-framework
+        run: ./gradlew jspecifySamplesTest --include-build ../jspecify
+# If a cloned EISOP CF is needed, use the following:
+#        run: ./gradlew jspecifySamplesTest --include-build ../jspecify --include-build ../checker-framework
 
   publish-snapshot:
     name: Publish Conformance Test Framework Snapshot

--- a/settings.gradle
+++ b/settings.gradle
@@ -13,7 +13,7 @@ exec {
 dependencyResolutionManagement {
     versionCatalogs {
         libs {
-            version("checkerFramework", "3.42.0-eisop4")
+            version("checkerFramework", "3.42.0-eisop5")
 
             library("checkerFramework-checker", "io.github.eisop", "checker").versionRef("checkerFramework")
             library("checkerFramework-checker-qual", "io.github.eisop", "checker-qual").versionRef("checkerFramework")

